pkgs/development/tools/misc/ccls/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, makeWrapper
+, cmake, llvmPackages, rapidjson }:
+
+stdenv.mkDerivation rec {
+  name    = "ccls-${version}";
+  version = "0.20181225.6";
```

Comment:
Sorry but if it didn't bother you much, could you please use the tag 0.20181225.7 ?

0.20181225.6 may cause unpleasant log lines in vscode and coc.nvim. https://github.com/MaskRay/ccls/issues/205